node/src/server-modules/GlideFt.ts

```diff
@@ -269,17 +268,73 @@ export class GlideFt {
             >
         ).then(convertGlideRecordToRecord);
     }
+
+    /**
+     * Parse a query and return information about how that query was parsed.
+     *
+     * @param client - The client to execute the command.
+     * @param indexName - The index name.
+     * @param query - The text query to search. It is the same as the query passed as
+     * an argument to {@link search | FT.SEARCH} or {@link aggregate | FT.AGGREGATE}.
+     * @param options - (Optional) See {@link DecoderOption}.
+     * @returns A query execution plan.
+     *
+     * @example
+     * ```typescript
+     * console.log(GlideFt.explain(client, "myIndex", "@price:[0 10]")); // Output:
+     * // "Field {\n\tprice\n\t0\n\t10\n}"
```

Comment:
Might look better if Output is just on the second line instead of splitting it across 2 lines.